integration_tests/pom.xml

```diff
@@ -180,6 +180,36 @@
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>exec-maven-plugin</artifactId>
                 <executions>
+                    <execution>
+                        <id>clean previously populated config file</id>
+                        <phase>verify</phase>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <configuration>
+                            <executable>rm</executable>
+                            <arguments>
+                                <argument>-f</argument>
+                                <argument>${spark.rapids.source.basedir}/${rapids.module}/target/spark-rapids-default-configs.json</argument>
+                            </arguments>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>populate default configs for testing</id>
+                        <phase>verify</phase>
+                        <goals>
+                            <goal>java</goal>
+                        </goals>
+                        <configuration>
+                            <includeProjectDependencies>true</includeProjectDependencies>
+                            <mainClass>com.nvidia.spark.rapids.tests.DumpDefaultConfigs</mainClass>
+                            <classpathScope>provided</classpathScope>
```

Comment:
Will it not work with the default [classpathScope](https://www.mojohaus.org/exec-maven-plugin/java-mojo.html#classpathScope) ?

Reply:
No, it doesn't. I guess it's because the `rapids-4-spark_*` dependency is `provided`.